modules/virtualization/microvm/appvm.nix

```diff
@@ -67,6 +65,9 @@
             pkgs.waypipe
           ];
 
+          # Add root user only for debug builds
+          users.users.admin.extraGroups = lib.mkIf config.ghaf.profiles.debug.enable ["wheel"];
```

Comment:
you already set this in the Groups.nix definitions

Reply:
Yes right we set [here](https://github.com/tiiuae/ghaf/pull/459/files#diff-b453010599f021b67ea495c89465afc98bd4679f9e7ad5333890a1dd1ec9cc86R21), the catch is now we have additional condition along with it `lib.mkIf config.ghaf.profiles.debug.enable` due to which admin will not be part of wheel group unless explicitly [added](https://github.com/tiiuae/ghaf/pull/459/files#diff-b7c4d8b1828ec4e55c3b85b348f4c180560550100c49b79a826d4cacfd1afaedR71) in respective app and system vm's.